floatsign.sh

```diff
@@ -280,6 +280,34 @@ then
 	fi
 fi
 
+# Check for and resign any embedded apple watch extensions (new feature for iOS 8 and above apps)
+PLUGINS_DIR="$TEMP_DIR/Payload/$APP_NAME/PlugIns"
+if [ -d "$PLUGINS_DIR" ];
+then
+  if [ "$TEAM_IDENTIFIER" == "" ];
+  then
+    echo "ERROR: embedded plugin detected, re-signing iOS 8 (or higher) applications wihout a team identifier in the certificate/profile does not work" >&2
+    exit 1;
+  fi
+  
+  echo "Resigning embedded plugins using certificate: '$CERTIFICATE'" >&2
+  for plugin in "$PLUGINS_DIR"/*
+  do
+    if [[ "$plugin" == *.appex ]]
+    then
+      for app in "$plugin"/*.app
+      do
+        /usr/bin/codesign -f -s "$CERTIFICATE" --entitlements="$ENTITLEMENTS" "$app"
```

Comment:
I finally decided to get rid of floatsign.sh since it starts to be too difficult to sign correctly an application. Shenzhen is really helpful for that.